src/main/java/org/logstash/beats/Server.java

```diff
@@ -29,11 +28,14 @@
     static final long SHUTDOWN_TIMEOUT_SECONDS = 10;
 
     private int port;
```

Comment:
not needed for this PR, but maybe all of these can be set in the constructor and made `final`?